packages/react-vapor/src/components/modal/tests/ModalComposite.spec.tsx

```diff
@@ -72,6 +72,24 @@ describe('ModalComposite', () => {
         expect(screen.getByRole('dialog', {name: 'second modal title'})).toBeVisible();
     });
 
+    it('finds the modal even if the modal title changes', async () => {
+        const ModalFixture = () => {
+            const [isLoading, setIsLoading] = React.useState(true);
+            React.useEffect(() => {
+                const timer = setTimeout(() => {
+                    setIsLoading(false);
+                }, 2000);

```

Comment:
Can you make the timeout shorter so the test run faster?

Reply:
woops that wasn't supposed to be committed, my bad